doc/source/reshaping.rst

```diff
@@ -518,6 +518,31 @@ the prefix separator. You can specify ``prefix`` and ``prefix_sep`` in 3 ways
     from_dict = pd.get_dummies(df, prefix={'B': 'from_B', 'A': 'from_A'})
     from_dict
 
+Sometimes it will be useful to only keep k-1 levels of a categorical
+variable to avoid collinearity when feed the result to statistical models.
+You can switch to this mode by turn on ``remove_first``.
+
+.. ipython:: python
+
+    s = pd.Series(list('abcaa'))
+
+    pd.get_dummies(s, remove_first=True)
+
+    df = pd.DataFrame({'key': list('bbacab'), 'data1': range(6)})
+
+    pd.get_dummies(df['key'], remove_first=True)
+
+You can also use ``remove_first`` as a equivalent to the ``omit.constants``
```

Comment:
you are using a term ``omit.constants`` very few prob know. keep it simple or explain.